### PR TITLE
delete all speakers of all lists of speakers

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts [hunk 1]
```diff
@@ -342,4 +342,13 @@ export class AgendaListComponent extends BaseListViewComponent<ViewItem> impleme
             return result;
         }
     }
+
+    public async deleteAllSpeakersOfAllListsOfSpeakers(): Promise<void> {
+        console.log("hi");
```
Comment (FinnStutzenstein) [inline client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts:347]: @tsiegleauq see email

File: client/src/app/site/agenda/components/agenda-list/agenda-list.component.html [hunk 2]
```diff
@@ -119,6 +119,11 @@ <h2>{{ 'Agenda' | translate }}</h2>
             <span>{{ 'Import' | translate }}</span>
         </button>
         <mat-divider></mat-divider>
+        <button mat-menu-item *osPerms="'agenda.can_manage'" class="red-warning-text" click="deleteAllSpeakersOfAllListsOfSpeakers()">
```
Comment (tsiegleauq) [inline client/src/app/site/agenda/components/agenda-list/agenda-list.component.html:122]: it's `(click)="deleteAllSpeakersOfAllListsOfSpeakers()">`

Comment (FinnStutzenstein) [reply client/src/app/site/agenda/components/agenda-list/agenda-list.component.html:122]: OMG :D I needed a break lol
Thanks

**Comment by FinnStutzenstein** [comment]
Waiting for #5326

**Pull request merged**